<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Doctor_record</class>
 <widget class="QDialog" name="Doctor_record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>361</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>40</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>12</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>14</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>101</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>331</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string>Вибрати</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>331</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string>Видалити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string>Знайти записи</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>13</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Лікар</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>334</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Час</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>351</y>
     <width>521</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>71</y>
     <width>521</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>10</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
     min-width: 80px;
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string>Вибрати</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Doctor_record</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Doctor_record</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
